### ISSUE TITLE
expect package outdated (1.13.0 --> 22.0.3)

### ISSUE THREAD

**Issue by ComFreek** [IonicaBizau/tester#15]
In a PR for [scrape-it](https://github.com/IonicaBizau/scrape-it), I'd like to use `expect(somehowBuildPromise()).resolves.toEqual(...)` in a test. However, [.resolves](https://facebook.github.io/jest/docs/en/asynchronous.html#resolves-rejects) is only available in expect 20.0.0+. (Note that the NPM expect package has been merged with and is now part of Facebook's Jest multitude of packages, see https://github.com/facebook/jest/issues/1679.)

I have tried simply updating expect, but the new version already breaks [tester's test](https://github.com/IonicaBizau/tester/blob/master/test/index.js) because `toBeA` is not existent (anymore?) in Jest's expect. So upgrading expect would require bumping tester's major version number.

**Comment by IonicaBizau** [comment]
Interesting! Except the `toBeA` are the other things working? I see the `expect` npm package has no content anymore...

**Comment by ComFreek** [comment]
> I see the `expect` npm package has no content anymore...

Yeah, its [NPM site](https://www.npmjs.com/package/expect) confused me as well! You can find its code here: https://github.com/facebook/jest/tree/master/packages/expect.

For example, `toEqual` works fine for me in a clean directory with `npm install expect`, its code is here: https://github.com/facebook/jest/blob/0ce82e6365feb1012a1dd1a6ae230a8012d90679/packages/expect/src/matchers.js#L416
I also saw in the meantime that `toBeA` is a won't fix: https://github.com/facebook/jest/issues/2990

**Comment by IonicaBizau** [comment]
7 years and 5 months later... 😂
